subsys/zigbee/osif/zb_nrf_platform.c

```diff
@@ -265,6 +267,7 @@ static int zigbee_init(struct device *unused)
 	ZB_SET_TRAF_DUMP_OFF();
 #endif /* ZB_TRACE_LEVEL */
 
+#ifndef CONFIG_ZB_TEST_MODE
```

Comment:
Consider moving definitions:
```	
        zb_ieee_addr_t ieee_addr;
	zb_uint32_t channel_mask;
```

below this line.
It would reduce two blocks `#ifndef CONFIG_ZB_TEST_MODE - #endif` to one.

Reply:
Changed.